foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/utils/SPIServiceUtils.java

```diff
@@ -149,7 +149,7 @@ private SPIServiceUtils() {
     List<T> services = getOrLoadSortedService(serviceType);
     return (IMPL) services
         .stream()
-        .filter(service -> service.getClass().equals(implType))
+        .filter(service -> implType.isAssignableFrom(service.getClass()))
```

Comment:
design purpose is equal
why do this change?

Reply:
I have updated the JIRA status. This PR is not valid. Please close it.